eng/Versions.props

```diff
@@ -11,14 +11,14 @@
     <!--
         When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
     -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
     <!--
         When DotNetFinalVersionKind is set to 'release' (only for the release branches),
         the build will produce stable outputs for 'Shipping' packages.
 
         This is used by the Arcade SDK (Publish.proj) to determine if the build is a release build or not.
       -->
-    <DotNetFinalVersionKind />
+    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment:
Are these changes expected?

Reply:
Yes, we will do another release from this branch, and we want package stabilization on release/9.5 branch to remain on 😃